config/solr.yml

```diff
@@ -4,6 +4,6 @@ development:
 test:
   url: <%= ENV.fetch('SCHOLARSARCHIVE_SOLR_URL', 'http://localhost:8985') %>/solr/hydra-test
 production:
-  url: <%= ENV.fetch('SCHOLARSARCHIVE_SOLR_URL', 'http://localhost:8080/solr/hydra-production') %>
+  url: <%= ENV.fetch('SCHOLARSARCHIVE_SOLR_URL', 'http://localhost:8080/solr/production') %>
```

Comment:
Is this production safe?

Reply:
yes, production sets that environment key properly. I did change the url to match the production SOLR core name (hydra-production -> hydra-prod)